Taskfile.yml

```diff
@@ -3,70 +3,63 @@
 
 version: "3"
 
-env:
-  DOCKER_IMAGE: cogito
-
 vars:
   DATE: '{{ now | date "2006-01-02" }}'
   COMMIT:
     sh: git log -n 1 --format=%h
   REPO: github.com/Pix4D/cogito
-  TAG:
+  DOCKER_TAG:
     sh: ci/git-ref-to-docker-tag.sh
-  BUILD_INFO: "Tag: {{.TAG}}, commit: {{.COMMIT}}, build date: {{.DATE}}"
```

Comment:
There is no need to modify this script `ci/git-ref-to-docker-tag.sh` ? At least removing the `do_travis()` would be nice.

Reply:
That script will go away in a subsequent PR, where I will remove all Travis-related code.